drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/TestToolsPresenter.java

```diff
@@ -283,6 +284,12 @@ public void onPerfectMatchSearchedEvent(String search, boolean notEqualsSearch)
                 .filter(entry -> filterTerm(entry.getKey(), search, notEqualsSearch))
                 .forEach(filteredEntry -> addSimpleJavaInstanceListGroupItemView(filteredEntry.getKey(), filteredEntry.getValue()));
         updateSeparators();
+        /* If notEqualsSearch is TRUE, then the instance is not assigned for the selected column. Therefore, it isn't
+           necessary to search through the maps to check it.
+         */
+        if (!notEqualsSearch) {
+            checkInstanceIsAssigned(search);
```

Comment:
Could we update the method name to say we are as checking so doing an action inside? Or return just boolean from the current and do the action according the result at this place? What do you think? Do we mix two things together or not? I may be wrong.

Reply:
I my opinion, it's better to change the name only. In this context, I see the search and the assignation as an atomic action, there is no need to perform a search without assign the result to the related view. For the name what do you suggest?

Reply:
@yesamer 
If this is the topic, I agree with @jomarko 
"updateInstanceIsAssignedStatus" -> IMO better reflect the scope of the method - "check" by itself does not mean "change something"